static/chats.js

```diff
@@ -156,16 +182,43 @@ function saprotiKomandu(teksts) {
 }
 
 function uzstadiVaardu(jaunaisVards) {
-  const vecaisVards = vards
-  setCookie('name', jaunaisVards, 90)
-  vards = jaunaisVards
+  const vecaisVards = vards;
+  vards = jaunaisVards;
   return `${vecaisVards} kļuva par ${vards}`
 }
 
+function nolasiGarastavokli(gStavoklis){
+  const gStavoklisUzServeri = new GaraStavoklis(vards, gStavoklis.join(' '))
```

Comment:
Kāpēc ne šādi?:
```js
const gStavoklisUzServeri = { 'vaards': vards, 'garaStaavoklis': gStavoklis }

Reply:
Jā, tad varu dzēst to klasi ārā un objekts nosūtīšanai uz serveri ir gatavs!